pygments/lexers/configs.py

```diff
@@ -1114,54 +1114,171 @@ class AugeasLexer(RegexLexer):
+            # strings as strings though.
+            # Start of bare key (only ASCII is allowed here).
+            (r'[A-Za-z0-9_-]+', Keyword),
+            (r'"', String.Double, 'basic-string'),
+            (r"'", String.Single, 'literal-string'),
+            (r'\.', Keyword),
+            (r'\]\]?', Keyword, '#pop'),
+
+            # Inline whitespace allowed
+            (r'[ \t]+', Whitespace),
+        ],
+        'value': [
+            # Datetime, baretime
+            (_datetime, Literal.Date, '#pop'),
+            (_time, Literal.Date, '#pop'),
 
-            # Operators
-            (r'=', Operator)
+            # Recognize as float if there is a fractional part
+            # and/or an exponent.
+            (r'[+-]?\d[0-9_]*[eE][+-]?\d[0-9_]*', Number.Float, '#pop'),
+            (r'[+-]?\d[0-9_]*\.\d[0-9_]*([eE][+-]?\d[0-9_]*)?',
+             Number.Float, '#pop'),
 
-        ]
+            # Infinities and NaN
+            (r'[+-]?(inf|nan)', Number.Float, '#pop'),
+
+            # Integers
+            (r'-?0b[01_]+', Number.Bin, '#pop'),
+            (r'-?0o[0-7_]+', Number.Oct, '#pop'),
+            (r'-?0x[0-9a-fA-F_]+', Number.Hex, '#pop'),
+            (r'[+-]?[0-9_]+', Number.Integer, '#pop'),
+
+            # Strings
+            (r'"""', String.Double, ('#pop', 'multiline-basic-string')),
+            (r'"', String.Double, ('#pop', 'basic-string')),
+            (r"'''", String.Single, ('#pop', 'multiline-literal-string')),
+            (r"'", String.Single, ('#pop', 'literal-string')),
+
+            # Booleans
+            (r'true|false', Keyword.Constant, '#pop'),
+
+            # Start of array
+            (r'\[', Punctuation, ('#pop', 'array')),
+
+            # Start of inline table
+            (r'\{', Punctuation, ('#pop', 'inline-table')),
+        ],
+        'array': [
+            # Whitespace, including newlines, is ignored inside arrays,
+            # and comments are allowed.
+            (r'\s+', Whitespace),
+            (r'#.*', Comment.Single),
+
+            # Delimiters
+            (r',', Punctuation),
+
+            # End of array
+            (r'\]', Punctuation, '#pop'),
+
+            # Parse a value and come back
+            default('value'),
+        ],
+        'inline-table': [
+            # Note that unlike inline arrays, inline tables do not
+            # allow newlines or comments.
+            (r'[ \t]+', Whitespace),
+
+            # Keys
+            include('key'),
+
+            # Values
+            (r'(=)(\s*)', bygroups(Punctuation, Whitespace), 'value'),
+
+            # Delimiters
+            (r',', Punctuation),
+
+            # End of inline table
+            (r'\}', Punctuation, '#pop'),
+        ],
+        'basic-string': [
+            (r'"', String.Double, '#pop'),
+            include('escapes'),
+            (r'[^"\\]+', String.Double),
+        ],
+        'literal-string': [
+            (r".*'", String.Single, '#pop'),
```

Comment:
I think this is a bug. This will capture too much if there is a literal string followed by a comment containing `'`. This broke mypy docs build on this line:
```
              'two\.pyi$',  # but TOML's single-quoted strings do not
```
see https://github.com/python/mypy/actions/runs/6916132945/job/18815845352

Reply:
Thanks, NP!

Reply:
Oh dear. I'll try to get it done today, somehow. I'm always afraid this happens, and our release process is still fairly manual :( Goals for 2024 I guess.

Reply:
Let me know if you want me to step in. Also, what would you like to automate?

Reply:
Things I'd like to automate:

* From tag to PyPi -- ideally, to test-pypi on every tagged commit (https://github.com/marketplace/actions/pypi-publish) -- ideally, the actual publication would be a special action I just click on. It's not that it takes a lot of time, but I'm always nervous I mess something up with the command line, forget to delete a file, git clean, etc. -- I very diligently work through the `release-checklist` to avoid that. Literally signing things off.
* Auto-formatting -- I tend to clean up the formatting of the lexers every time close to release, at least the worst offenders. I use `autopep8` at the moment, would rather apply `flake8` or `black` on the entire codebase.
* Auto-check that the new arguments like URL etc. are present on new lexers
* Auto-check `.. versionadded:: ` is there -- costs me _a lot of time_ to open up every Lexer close to release and make sure it's present and in the right format (i.e. `2.17.0` vs. `2.17`)
* Actually get all checks working/passing (i.e. the additional checkers I wrote and possibly PyLint). `check_whitespace_tokens` and `check_repeated_tokens` need an expected-fail list so we can whitelist currently existing lexers until we fix those, but new lexers should always pass those tests.
* Verify all PR numbers closed/merged since last release are mentioned in the `CHANGES` file. I'm pretty good at assigning tasks to milestones now, but I still miss things in the `CHANGES` file, and it's super time consuming to open 100 tabs, go through each item one-by-item, check the PR number/issue number is present, etc. If there was a way to auto-generate the changelog that would be even better, but my experience is that those look pretty ugly and some manual checkup is fine.

I'll get to the release in a moment, thanks for the offer though!

Reply:
Thank you for the quick fix! :+1: